packages/ng/src/app/tree-picker/standalone-tree/tree.component.html

```diff
@@ -0,0 +1,5 @@
+<ul>
```

Comment:
could the tree-item be enough ? As it is alway reccursive

Reply:
I don't think so: the `lu-tree` component is in charge of managing the ngModel, whereas the `lu-tree-item` components only manipulate the internal `tree` attribute